Server/mods/deathmatch/logic/CAccount.cpp

```diff
@@ -289,6 +289,15 @@ bool CAccount::RemoveSerial(const SString& strSerial)
     return false;
 }
 
+//
+// Replace the serial number for a specific account
+//
+void CAccount::SetAccountSerial(const std::string strSerial)
```

Comment:
Please avoid using Hungarian notation in new code, `serial` would be neater

Reply:
adjusted 9ad1188